homeassistant/components/easyenergy/__init__.py

```diff
@@ -25,7 +27,7 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     return True
 
 
-async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+async def async_setup_entry(hass: HomeAssistant, entry: EasyEnergyConfigEntry) -> bool:
     """Set up easyEnergy from a config entry."""
 
     coordinator = EasyEnergyDataUpdateCoordinator(hass)
```

Comment:
It might also be a good time to start passing the entry explicitly to the coordinator:
```suggestion
    coordinator = EasyEnergyDataUpdateCoordinator(hass, entry)
```